### ISSUE TITLE
Can't run functions locally

### ISSUE THREAD

**Issue by cluda** [serverless/serverless#325]
Getting 'TypeError: Cannot read property 'lambda' of undefined' when trying to run a function locally.

```
➜ serverless project create
 _______                             __
|   _   .-----.----.--.--.-----.----|  .-----.-----.-----.
|   |___|  -__|   _|  |  |  -__|   _|  |  -__|__ --|__ --|
|____   |_____|__|  \___/|_____|__| |__|_____|_____|_____|
|   |   |             The Serverless Application Framework
|       |                          serverless.com, v.0.0.6
`-------'
Serverless: Enter a project name:  (serverless-4kdZZqyHg)
Serverless: Enter a project domain (used for serverless regional bucket names):  (myapp.com)
Serverless: Enter an email to use for AWS alarms:  (me@myapp.com)
Serverless: Select a region for your project:
    us-east-1
  > us-west-2
    eu-west-1
    ap-northeast-1
Serverless: Select an AWS profile for your project:
    default
  > jaws-admin
    eb-cli
Serverless: Creating a project region bucket on S3: serverless.uswest2.myapp-41fzzqjs.com...
Serverless: Creating CloudFormation Stack for your new project (~5 mins)...
Serverless: Successfully created project: serverless-4kdZZqyHg
➜ cd serverless-4kdZZqyHg
➜ serverless module create
Serverless: Enter a name for your new module:  module1
Serverless: Enter a function name for your new module:  func1
Serverless: Installing "serverless-helpers" for this module via NPM...
module1@0.0.1 /Users/sogasg/devTemp/serverless/serverless-4kdZZqyHg/back/modules/module1
└─┬ serverless-helpers-js@0.0.3
  └── dotenv@1.2.0

Serverless: Successfully created new serverless module "module1" with its first function "func1"
➜ cd back/modules/module1/func1
➜ serverless function run
/usr/local/lib/node_modules/serverless/node_modules/bluebird/js/main/promise.js:677
            throw e;
            ^

TypeError: Cannot read property 'lambda' of undefined
    at FunctionRun.functionRun (/usr/local/lib/node_modules/serverless/lib/actions/FunctionRun.js:65:32)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
From previous event:
    at /usr/local/lib/node_modules/serverless/lib/Serverless.js:207:25
    at Array.reduce (native)
    at Serverless.actions.(anonymous function) (/usr/local/lib/node_modules/serverless/lib/Serverless.js:206:25)
    at Serverless.command (/usr/local/lib/node_modules/serverless/lib/Serverless.js:180:44)
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/bin/serverless:13:36)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
➜
```

**Comment by yaochen** [comment]
same here.

on node v4.2.1

**Comment by michaelwills** [comment]
@yaochen see a temporary workaround here:

https://github.com/serverless/serverless/issues/326#issuecomment-162833926

but the API Gateway endpoint isn't created just yet.

**Comment by yaochen** [comment]
@michaelwills 

nice! curious, is that something that was mistakenly left out or this is a new bug in Serverless?

**Comment by michaelwills** [comment]
@yaochen 
Heh well Serverless was JAWS up until yesterday and went through a substantial rewrite so there are a number of bits still broken as the refactor isn't 100% yet. Join the gitter channel and scroll back through the history of the last 2 days. :)

**Comment by yaochen** [comment]
understood. just overly excited to try out the new jaws :)

**Comment by michaelwills** [comment]
Heh I know what you mean. @ac360 noted a minute ago they are planning to fix a lot today so we'll see what a day may bring.

**Comment by ac360** [comment]
Working on this now.  We made a big change to the Function JSON format at the last minute and it broke this.  Should be fixed in an hour :)

**Comment by ac360** [comment]
Should be fixed in 99e5856dbca6a80c3b1744acd70a65943ccf6fb4.  Writing tests for it right now.  We will push out a new `npm` version tonight (after we make more fixes) which will feature this.  Thanks for your help testing @yaochen @cluda @michaelwills
